src/graphorm/index.ts

```diff
@@ -23,7 +24,6 @@ import { GQLBookmarkList } from '../schema/bookmarks';
 import { base64 } from '../common';
 import { GQLComment } from '../schema/comments';
 import { GQLUserPost } from '../schema/posts';
-import { UserNotification } from '../entity/notifications/UserNotification';
```

Comment:
Just a cleaner import I noticed.
See above the `schema` ones, we should probably add a linter here for cleaner imports as we have on apps project.